### PR TITLE
[WFLY-18461] cmt Quickstart Common Enhancements CY2023Q3

### PR DESCRIPTION
**Pull request by marcosgopen**
Issue: https://issues.redhat.com/browse/WFLY-18461

### CODE REVIEW

**Review by emmartins — CHANGES_REQUESTED** [review]
Besides the pointed changes please also change runtimeName at https://github.com/wildfly/quickstart/pull/731/files#diff-2013e1734bf0fce09cc1ce9a533ea791cc7e3737fcd82b8d8a0d85fc1caff0efL234 , to name. As shown now at https://docs.google.com/document/d/1GFnZrnlwRkwC0HfziRws2UUWR99KdaQayA7Ei12SGN4/edit#heading=h.ob4izb19fxpk

File: cmt/pom.xml [hunk 1]
```diff
@@ -306,26 +248,22 @@
...
                             </includes>
-                            <excludes>
-                                <exclude>**/ProvisionedManagedCustomerIT</exclude>
-                            </excludes>
                         </configuration>
                         <executions>
                             <execution>
                                 <goals>
                                     <goal>integration-test</goal>
-                                    <goal>verify</goal>
```
[the remark this answers is not in the document]
Comment (marcosgopen) [reply cmt/pom.xml]: Ok, I've just fixed it.

File: .github/workflows/quickstart_cmt_ci.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'cmt/**'
+      - '.github/workflows/quickstart_ci.yml'
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: cmt
+      MICROPROFILE: false
```
[the remark this answers is not in the document]
Comment (marcosgopen) [reply .github/workflows/quickstart_cmt_ci.yml]: Ok, I 've just replaced       _MICROPROFILE: false_ with _TEST_PROVISIONED_SERVER: true_

**Comment by marcosgopen** [comment]
> Besides the pointed changes please also change runtimeName at https://github.com/wildfly/quickstart/pull/731/files#diff-2013e1734bf0fce09cc1ce9a533ea791cc7e3737fcd82b8d8a0d85fc1caff0efL234 , to name. As shown now at https://docs.google.com/document/d/1GFnZrnlwRkwC0HfziRws2UUWR99KdaQayA7Ei12SGN4/edit#heading=h.ob4izb19fxpk

Thanks @emmartins , I addressed you review.

**Review by emmartins — APPROVED** [review]
thanks @marcosgopen, all looks good to me

**Comment by emmartins** [comment]
@kabir FYI this one is ready for OpenShift testing too

**Pull request merged**